index.html

```diff
@@ -18,42 +18,277 @@
+					text ever since the 1500s, when an <br>
+					unknown printer took a standard dummy <br>
+					text.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/CSS.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/html.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/bootstrap.png"></a>
+					</li>
+					<li>
+						<a href="#"><img src="Works-Images/Ruby.png"></a>
+					</li>
+				</ul>
+				<button type="button" class="btn">See project</button>
+			</div>
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
```

Comment:
Your design
![image](https://github.com/hamidazim321/Portfolio-Project/assets/78251927/3cad39a2-39e6-456a-9037-5ba9822ee92c)

Kindly make technology in each category(e.g., "HTML", "Bootstrap" and "Ruby") a list `ul` and each of them is a `li`